packages/web/src/ui/flow/design.js

```diff
@@ -103,7 +103,7 @@ const reducers = {
  * @returns {Object} the updated state
  */
   setDesignContent: (state, payload) => {
-    console.log('design: set content', state.design, payload)
+    console.log('design: set content', state, state.design, payload)
```

Comment:
I think console messages should be commented out for the final PR.

Reply:
agreed, but that should be done at a later stage once we are truely in cleanup mode ;)